amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2091,7 +2094,148 @@ paths:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventsInfo'
+        '401':
+          description: Authentication failed
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+
+    post:
+      tags:
+        - events
+      summary: Create a new records management event
+      description: |
+        Create a new records management event.
+      operationId: createEvent
+      parameters:
+        - in: body
+          name: eventBodyCreate
+          description: The event information to create. The event type value can be one of the following- [ Simple, Obsoleted, Superseded, Cross-Referenced Record Transferred, Versioned ]
+          required: true
+          schema:
+            $ref: '#/definitions/EventBodyCreate'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '201':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventEntry'
+        '400':
+          description: Cannot create event. An event with the name **name** already exists  
+        '401':
+          description: Authentication failed
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  
+  '/events/{eventId}':
+    get:
+      tags:
+        - events
+      summary: Return event for given eventId
+      description: |
+        Get an record management event information using eventId
+      operationId: getEvent
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/eventIdParam'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventInfo'
+        '401':
+          description: Authentication failed
+        '404':
+          description: "**eventId** does not exist"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    put:
+      tags:
+        - events
+      summary: Update event for given eventId
+      operationId: updateEvent
+      description: |
+        Update record management event information using eventId. The event type value can be one of the following- [ Simple, Obsoleted, Superseded, Cross-Referenced Record Transferred, Versioned ]
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/eventIdParam'
+        - in: body
+          name: eventBodyUpdate
+          description: The event information to update.
+          required: true
+          schema:
+            $ref: '#/definitions/EventBodyUpdate'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventInfo'
+        '401':
+          description: Authentication failed
+        '400':
```

Comment:
I don't know whether it matters, but everywhere else we list the status codes in numerical order.

Reply:
Done